Simplenote/AccountVerificationRemote.swift

```diff
@@ -41,6 +41,7 @@ class AccountVerificationRemote {
         var request = URLRequest(url: verificationURL.appendingPathComponent(base64EncodedEmail),
                                  cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
                                  timeoutInterval: RemoteConstants.timeout)
+        request.setValue(Locale.current.languageCode, forHTTPHeaderField: "Accept-Language")
```

Comment:
I don't think you need to manually set it.

To test I commented the code to set `Accept-Language` header and added this snippet to check which headers are sent:
```
        dataTask.resume()
        DispatchQueue.main.async {
            print(dataTask.currentRequest?.allHTTPHeaderFields)
        }
```

And the result is:
```
allHTTPHeaderFields : Optional<Dictionary<String, String>>
      ▿ some : 5 elements
        ▿ 0 : 2 elements
          - key : "Content-Type"
          - value : "application/json"
        ▿ 1 : 2 elements
          - key : "Content-Length"
          - value : "21"
        ▿ 2 : 2 elements
          - key : "Accept-Encoding"
          - value : "gzip, deflate, br"
        ▿ 3 : 2 elements
          - key : "Accept-Language"
          - value : "en-gb"
        ▿ 4 : 2 elements
          - key : "Accept"
          - value : "*/*"

```

Reply:
SO MUCH better!! Thank you!!